dashboard/app/controllers/lessons_controller.rb

```diff
@@ -132,6 +132,8 @@ def update
   def clone
     destination_script = Script.find_by_name(params[:destinationUnitName])
     raise "Cannot find script #{params[:destinationUnitName]}" unless destination_script
+    raise 'Destination script and lesson script must be in a course version' unless destination_script.get_course_version && @lesson.script.get_course_version
```

Comment:
Consider maybe moving to unit instead of script in the language and variables throughout this PR?

Reply:
I'm open to opinions here, but my personal perspective is that having a mix of terminology causes more pain than the pain of having to find/replace in a few more places. Based on that I'd rather use the current terminology at least until we've moved the models over to the new terminology.

Reply:
I would say we are already using a mix of unit and script in different places so I don't think we are preventing confusion that much by avoiding it. Also renaming script to unit is listed as part of Q2 work so seems like it helps get us toward that goal. Doing all the find and replace and untangling what can be renamed takes time and effort :) I've been doing it for almost 3 weeks now for lessons

Reply:
OK, good points. Either way is fine with me.

Reply:
Considering we did move to unit on the front-end of this feature, so we're already mixing the terminology, I'll change to unit here

Reply:
I'm going to update the checks but I'll update `destination_script` in a follow-up PR as I don't want to clutter this PR too much.